graphql/resolvers.ts

```diff
@@ -195,10 +202,16 @@ const resolvers = {
           );
           const final_amount = amount - fees_result.data.onChainTxFee.amount;
           if (final_amount <= 0) {
-            throw new Error("Amount is less than fees");
+            throw new CustomError(
```

Comment:
we currently don't throw for mutation in our main backend. instead of we return an object that contain an error. we should do the same here.

Reply:
> we currently don't throw for mutation in our main backend. instead of we return an object that contain an error. we should do the same here.

I believe we should create a separate issue for this. I need to focus on both the resolver and error handling aspects.